LICENSE

```diff
@@ -2,5 +2,6 @@ This software is made available under the terms of *either* of the licenses
 found in LICENSE.APACHE or LICENSE.BSD. Contributions to cryptography are made
 under the terms of *both* these licenses.
 
-The code used in the OpenSSL locking callback is derived from the same in
-Python itself, and is licensed under the terms of the PSF License Agreement.
+The code used in the OpenSSL locking callback and OS random engine is derived
+from the same in Python itself, and is licensed under the terms of the PSF
```

Comment:
Ugh, nit, not technically a bug in your PR (but if I go fix it separately it'll just cause a conflict). Can you `s/Python/CPython/`